src/main/java/com/gmail/goosius/siegewar/utils/SiegeWarMoneyUtil.java

```diff
@@ -407,4 +410,43 @@ public static void payUpfrontSiegeStartCost(Siege siege) {
 		}
 	}
 
+	/**
+	 * Calculate the estimated amount of money in the economy.
+	 * 
+	 * The result is stored in this class.
+	 * 
+	 * Result = (All money in town banks + All money in nation banks)
+	 *          +10% (an estimate of how much else residents are carrying)
+	 * 
+	 * @param siegeWarPluginError true if SW is in error.
+	 */
+	public static void calculateEstimatedTotalMoneyInEconomy(boolean siegeWarPluginError) {
+		if(siegeWarPluginError) {
+			SiegeWar.severe("SiegeWar is in safe mode. Money calculation not attempted.");
+			return;
+		}
+		double result = 0;
+		//Town Accounts
+		for(Town town: TownyAPI.getInstance().getTowns()) {
+			result += town.getAccount().getHoldingBalance();
+		}
+		//Nation Accounts
+		for(Nation nation: TownyAPI.getInstance().getNations()) {
+			result += nation.getAccount().getHoldingBalance();
+		}
```

Comment:
It might be better to try and use the cached nation and town account balances. 

It should be tested to see if the caches are empty on startup or not.

Reply:
I tried that while developing this, but the numbers I got were much higher than expected.

Reply:
Did you do any debugging to see how the cached balance compared to the getHoldingBalance()?

You would have to call the cached one first. It doesn't make a lot of sense that it would be higher than the actual balance. If anything I would expect it to return 0.

Reply:
- Yea its weird. No I didn't really have time to debug.  I figured since the calculation is only done on server startup, it wouldn't matter so much either way.
- How about I create an issue ticket to look at this in the Streamlining release ?

Reply:
Ok done. I put this on async